frontend/src/layout/onboarding/onboarding.js

```diff
@@ -0,0 +1,162 @@
+        }
+    }, [])
+
+    async function dontShowAgain() {
+        await api.update('api/user', { onboarding: { ...user.onboarding, active: false } })
+        loadUser()
+    }
+
+    function content() {
+        return (
+            <div
+                data-attr="onboarding-content"
+                ref={contentRef}
+                style={{ display: 'flex', width: '25vw', flexDirection: 'column' }}
+            >
+                <h2>Get Started</h2>
+                <i>
+                    Complete these steps to learn how to use Posthog! Click on an item below to learn how to create one
+                </i>
+                {Object.entries(TourType).map(([, value], index) => {
+                    return (
+                        <div key={index}>
+                            <hr style={{ height: 3, visibility: 'hidden' }} />
+                            <Link
+                                onClick={() => {
+                                    closePopup()
+                                    setInstructionalModal(true)
+                                    setTourType(value)
+                                }}
+                                data-attr={'onboarding-item-' + index}
+                            >
+                                <Checkbox
+                                    style={{ marginRight: 12 }}
+                                    checked={user.onboarding.steps[index] || checked[index]}
+                                ></Checkbox>
+                                <span>{value}</span>
+                            </Link>
+                        </div>
+                    )
+                })}
+                <hr style={{ height: 5, visibility: 'hidden' }} />
+                {unfinishedCount > 0 ? (
+                    <p onClick={dontShowAgain} style={{ color: 'gray', cursor: 'pointer' }}>
+                        Don't show this again
+                    </p>
+                ) : (
+                    <Button onClick={dontShowAgain} type={'primary'}>
+                        Done
+                    </Button>
+                )}
+            </div>
+        )
+    }
+
+    return (
+        <div className="mr-2 ml-2">
+            <Popover
+                visible={visible}
+                content={actionsLoading ? <Loading></Loading> : content({ actions })}
+                trigger="click"
+            >
+                <Badge count={unfinishedCount}>
+                    <Button data-attr="onboarding-button" onClick={() => (visible ? closePopup() : setVisible(true))}>
+                        {unfinishedCount === 0 ? <StarFilled></StarFilled> : <StarOutlined></StarOutlined>}
+                    </Button>
+                </Badge>
+            </Popover>
+            <Modal
+                visible={instructionalModal}
+                style={{ minWidth: '50%' }}
+                bodyStyle={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}
+                footer={null}
```

Comment:
`footer` must be explicitly passed as `null`? that'd be a bit of an annoying API